FMC signatures

Ivan-Brigidano opened the pull request:
Become mandatary to have three signatures: the competitor's one, the judge's (usually a WCA Delegate) and the solution checker's one.

lgarron committed:
Recognize Mean of 3 for 4x4x4/5x5x5 BLD. Closes #539.

lgarron committed:
Style guide: add some more alternatives.

micahstairs committed:
Fix up grammar

micahstairs committed:
Improve wording

micahstairs committed:
Minor edits

AlbertoPdRF committed:
Change "time" to "result" in 9f13)

Thanks to Ron for pointing this out!

viroulep committed:
Update continent for Trinidad and Tobago

AlbertoPdRF committed:
Fix 9f14) and change "shorter" to "smaller".

viroulep committed:
Update last textual revision

jfly committed:
The Regulations are actually written with American English now.

See https://github.com/thewca/wca-regulations/issues/411.

lgarron committed:
Clarify that starting the solve ends inspection. (#579)

Two changes:
- Reword A4d) to be clear that the solve starts when the competitor starts the timer, regardless of whether they looked for a green light.
- Clarify that the competitor is not permitted to reset (and therefore is not permitted to "test" or "restart") the timer after starting it, even if they attempt to do so before they reach the what would have been the limit for inspection. Addresses #573.

lgarron committed:
Fix two inconsistent "scrambl{ing->e}" terms, and add a Guideline.

lgarron committed:
Explicitly forbid causing intentional puzzle defects. (#538)

lgarron committed:
Add `CODEOWNERS` file so that GitHub will prefer WRC members for review.

https://blog.github.com/2017-07-06-introducing-code-owners/

lgarron committed:
Update A6) to make it clear that stopping the timer always stops the solve. (#593)

lgarron committed:
Update A3c3++) to avoid implying that stopping the timer always results in a DNF. (#592)

It's only a DNF if the puzzle is not solved. The puzzle can't be solved if the
competitor hasn't done any moves (due to the scramble requirements in 4b3), but
version of A3c3++) introduced by fff14fe4 didn't specify that it was referring
to that case. This commit avoids directly talking about disqualification
directly in A3c3++), focusing instead on the fact that he solve was stopped.

pedrosino committed:
Clarify about darkest green (#597)

* Clarify about darkest green

* Add "adjacent" to the darkest green face.

lgarron committed:
Set a time limit on restoring missing signatures, and require Delegate discretion. (#606)

lgarron committed:
9f2) Change "nearest second" rounding examples to x.49 and x.50 (#613)

This makes the range from x.41 to x.49 extra-clear, and matches the precision
that most competition stopwatches have.

sarahstrong314 committed:
Fixed typo.

micahstairs committed:
Update event names throughout text to match official event names.

Some of the events were being referenced using inconsistent wording (in particular, the 3x3x3 Multi-Blind event). This change resolves many of those inconsistencies.

Fixes #582.

ronaldmansveld committed:
Add provison that MBLD is always measured in seconds

ronaldmansveld committed:
Update based on Oleg's feedback

lgarron committed:
Remove 3h2+, since stickerless puzzles have been permitted for nearly 3 years now. (#548)

lgarron committed:
2i) Clarify that interaction with the Stackmat timer is allowed, even though it is electronic. (#580)

This came up in #567.

lgarron committed:
Specify that there may be multiple Competitors Areas.

pedrosino committed:
Reword A6g and remove A6g+ (#557)

* Fix A6g

* Update wca-regulations.md

Co-Authored-By: pedrosino <pedrosino@gmail.com>

Baiqiang committed:
Change words for A4b1 (#645)

lgarron committed:
Consolidate time limit definitions in the Regulations. Closes #611. (#612)

* Consolidate time limit definitions in the Regulations. Closes #611.

This moves many details from the Guidelines to the Regulations. It also includes
two functional changes:

- Time limits must be announced before the competition, rather than just before the
  round.
- Only one cumulative time limit can apply to to a given round.

Both of these are almost always (or always?) followed in practice.

Note that this change "recycles" A1a2++) for a Guideline that isn't particular related to the old A1a2++. We don't usually do this, but it avoids creating a Guideline with the loooooong label of A2a++++++ and I don't foresee it causing issues here.

* Update wca-regulations.md

Co-Authored-By: lgarron <code@garron.net>

lgarron committed:
Refactor notation under Article 12 and make several FMC cleanups. (#643)

lgarron committed:
Tweak multi BLD event name, and distinguish "time" from the rest of the result.

lgarron committed:
Merge branch 'multi-blind-timing' into draft

lgarron committed:
Update H1d+/H1d++ to avoid implying that the multi BLD time is the full result. (#648)

lgarron committed:
Require the Delegate to restrict scramble access to groups before they start. (#651)

lgarron committed:
Specify that FMC pens may be self-supplied. (#647)

lgarron committed:
Specify that scramble signatures *should* be used. (#662)

* Specify that scramble signatures *should* be used. Addresses #640.

* (or initial)

lgarron committed:
Remove "Best of 1"/"Best of 2" as formats for 6x6x6 and 7x7x7. (#661)

Remove "Best of 1"/"Best of 2" as formats for 6x6x6 and 7x7x7. Closes #576.

lgarron committed:
Allow competitors to write their name while submitting an FMC solution. Closes #653. (#663)

AlbertoPdRF committed:
Merge pull request #672 from AlbertoPdRF/UpdateEventsNames

Update event names throughout text to match official event names.

lgarron committed:
Specify that the judge and Delegate may estimate cumulative elapsed time. (#669)

* Specify that the judge and Delegate may estimate cumulative elapsed time.

* Specify "the *WCA* Delegate" for consistency.

Co-Authored-By: lgarron <code@garron.net>

kingmathyall committed:
Merge branch 'draft' into big-cube-bld-means

kingmathyall committed:
Merge pull request #547 from thewca/big-cube-bld-means

Recognize Mean of 3 for 4x4x4/5x5x5 BLD. Closes #539.

pedrosino committed:
Require competitors to provide date consistent with the past. Closes #594

* Update 2c

* Fix serial comma

AlbertoPdRF committed:
Get rid of BoX formats in 9b1a). (#671)

Get rid of BoX formats in 9b1a).

Fixes #658.

lgarron committed:
Add a Guideline about visual isolation of the scramblers. (#676)

lgarron committed:
4b2++: Add a comma to make the sentence easier to understand.

lgarron committed:
Allow individual non-permitted puzzles in multi BLD to count as unsolved. (#670)

* Allow individual non-permitted puzzles in multi BLD to count as unsolved. Closes #545.

* Apply suggestions from review

Co-Authored-By: lgarron <code@garron.net>

lgarron committed:
Accidental pop assistance (#677)

* Clarify allowed assistance during pops. Closes #624.

* Update wca-guidelines.md

* Tweaks based on @kingmathyall's feedback.

AlbertoPdRF committed:
Move Regulations and Guidelines to the WCA Competition Requirements Policy. (#673)

Related issues: #553, #604, #610, #614, #629, #635, #641, #650.

Policy PR: https://github.com/thewca/wca-documents/pull/40

lgarron committed:
Allow competitors to touch stopwatches. #581 (#679)

This also has a provision for the organization team to provide the competitor an unofficial stopwatch instead.

lgarron committed:
Clarify that individual non-permitted puzzles must count as unsolved in the total. (Followup to #670) (#680)

* 3k2b+: Clarify that individual non-permitted puzzles in multi BLD count towards the # of puzzles.

* Clarify that an individual non-permtted puzzle in Multi BLD must not be replaced.

viroulep committed:
Merge branch 'official' into draft

lgarron committed:
Overhaul Fewest Moves (#678)

* Overhaul FMC Regulations

- Clean up the structure.
- Specify when the competitor may write their name:
  - Before the attempt.
  - While turning in a solution, in fron of the judge.
  - NOT after turning in a solution.
- Define an unambiguous, valid solution.
- Specify that a competitor should scribble out extra moves. (This is a
  "should", but placing it in the Regulations will hopefully encourage
  competitors to do this move consistently, leading to fewer frustrating calls
  while grading).

lgarron committed:
Specify a penalty in E2a1. (#689)

The default penalty is already DNF, but spelling it out is better.

lgarron committed:
Clarify that 9b specifies the valid combined rounds. (#692)

After #690 (Restore Multi-Blind combined rounds), this should make it unambiguous which combined round formats are allowed for which event.

lgarron committed:
Specify that competitors may join FMC late, if no one has left. (#691)

* Specify that competitors may join FMC late, if no one has left.

* Update wca-guidelines.md

Co-Authored-By: lgarron <code@garron.net>

lgarron committed:
Remove an extra space from Regulation 9g.

lgarron committed:
Revert "Remove an extra space from Regulation 9g."

This reverts commit 49d47ccd866a3ee2a2eaa41bdc165412014c9804.

lgarron committed:
Revert "Clarify that 9b specifies the valid combined rounds. (#692)"

This reverts commit ea29e0936c252281258711562d561f6128b08557.

lgarron committed:
Merge branch 'revert-exactly-specify-valid-combined-rounds' into draft

Mollerz committed:
Explicitly allow non-electronic hand warmers and food&drink (#701)

* Explicitly allow non-electronic hand warmers and food&drink

Instead of adding this to A5b it makes more sense to add them to 2i1. This parent regulation states already that it is at the discretion of the WCA delegate so it does not need to be included within 2i1c and 2i1d which are being added.

See: https://github.com/thewca/wca-regulations/issues/700

* 2i1c+) [CLARIFICATION]

lgarron committed:
Fewest Moves: Explicitly allow pencils and other writing instruments. Closes #696.

kingmathyall committed:
Merge pull request #705 from thewca/pencil

Fewest Moves: Explicitly allow pencils and other writing instruments.

lgarron committed:
Specify exactly which round formats are recognized. (#704)

* Specify exactly which round formats are recognized. Closes #693.

* Fix one round format.

* Update wca-regulations.md

Co-Authored-By: lgarron <code@garron.net>

* Update wca-regulations.md

Co-Authored-By: lgarron <code@garron.net>

* Update wca-regulations.md

Co-Authored-By: lgarron <code@garron.net>

pedrosino committed:
Change A1a to "must apply" (#698)

lgarron committed:
Allow the competitor to write their registrant ID for FMC. (#688)

This is important to specify because otherwise writing the registrant ID before
the attempt (E2a1) is a DNF.

See https://github.com/thewca/tnoodle/issues/379 for a TNoodle issue about
adding a registrant ID field to TNoodle FMC sheets.

xsrvmy commented:
I think this might cause scheduling issues and also forces the competitor to stay longer at the venue after the attempt

lgarron commented:
Hi @Ivan-Brigidano, I don't see any commits in this range, so I'm guessing you just meant to suggest this as a normal issue. I'll break this into a separate issue to avoid confusion.

Ivan-Brigidano commented:
Thanks you @lgarron  I'm new in Git so I made this little mistake, sorry.
The idea is to have signatures to make it more professional and have the certainty of who has corrected, judged... . I personally don't think it would suppose many time delays.
However, I don't compete in FMC and maybe I am suggesting something unnecessary.

lgarron committed:
Fix typo: `organization must apply` → `organization team must apply`.

Introduced in 7b6a685ae6774400303edd6626fff9166fbe40f3, which I merged and reviewed, so it's also my fault. :-P

Thanks to @OJCubing for noticing.
